### PR TITLE
Updated the celery config to add support for celery 5

### PR DESCRIPTION
**Pull request by mraarif**
Upgrade celery version to use celery 5 with tox

**JIRA:** 
https://openedx.atlassian.net/browse/BOM-2172

### CODE REVIEW

File: super_csv/celery.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+"""
+configuration file for celery
+"""
+
+from celery import Celery
+
+app = Celery('super-csv',
+             broker='memory://',
+             backend='db+sqlite:///:memory:',
```
Comment (awais786) [inline super_csv/celery.py:9]: It should come from settings.

File: super_csv/celery.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+             include=['super-csv.tasks'])
+
+app.conf.update(
+    task_always_eager=True,
+    task_eager_propagates=False,
+    CELERY_BROKER_HOSTNAME='localhost'
```
Comment (awais786) [inline super_csv/celery.py:15]: Same here

File: .travis.yml [hunk 3]
```diff
@@ -1,15 +1,18 @@
...
+  - TOXENV=django22-celery50-drf37
+  - TOXENV=django22-celery44-drf38
+  - TOXENV=django22-celery50-drf38
+  - TOXENV=django22-celery44-drf39
+  - TOXENV=django22-celery50-drf39
+  - TOXENV=django22-celery44-drflatest
+  - TOXENV=django22-celery50-drflatest
+  - TOXENV=quality
+  - TOXENV=docs
+  - TOXENV=pii_check
```
Comment (jmbowman) [inline .travis.yml:15]: Why the indentation change?  Not a big deal, but makes this inconsistent with the rest of the file.

File: super_csv/__init__.py [hunk 4]
```diff
@@ -2,6 +2,6 @@
 CSV Processor.
 """
 
-__version__ = '1.1.0'
+__version__ = '1.2.0'
```
Comment (jmbowman) [inline super_csv/__init__.py:5]: Technically dropping support for a Python version should be a major version bump (2.0.0), although I don't think we did that when we dropped 2.7 support.

File: Makefile [hunk 5]
```diff
@@ -40,9 +40,15 @@ upgrade: ## update the requirements/*.txt files with the latest packages satisfy
...
-	# Let tox control the Django version for tests
+	# Let tox control the versions for Django, DRF and celery for testing
 	sed -i.tmp '/^django==/d' requirements/test.txt
 	sed -i.tmp '/^djangorestframework==/d' requirements/test.txt
+	sed -i.tmp '/^amqp==/d' requirements/test.txt
+	sed -i.tmp '/^anyjson==/d' requirements/test.txt
+	sed -i.tmp '/^billiard==/d' requirements/test.txt
+	sed -i.tmp '/^celery==/d' requirements/test.txt
+	sed -i.tmp '/^kombu==/d' requirements/test.txt
+	sed -i.tmp '/^vine==/d' requirements/test.txt
```
Comment (jmbowman) [inline Makefile:51]: Shouldn't we also pin exact versions of all these in new `celery44.txt` and `celery50.txt` files like we've done elsewhere?

File: manage.py [hunk 6]
```diff
@@ -19,11 +19,11 @@
...
         try:
             import django  # pylint: disable=unused-import
-        except ImportError:
+        except ImportError as import_exception:
             raise ImportError(
                 "Couldn't import Django. Are you sure it's installed and "
                 "available on your PYTHONPATH environment variable? Did you "
                 "forget to activate a virtual environment?"
-            )
+            ) from import_exception
```
Comment (jmbowman) [inline manage.py:27]: Huh, I hadn't noticed this was added to Python 3.  Really need to go back and review all the release notes now that we're mostly on Python 3.8...

File: tox.ini [hunk 7]
```diff
@@ -9,6 +9,8 @@ deps =
     drf38: djangorestframework<3.9
     drf39: djangorestframework<3.10
     drflatest: djangorestframework
+    celery44: -r{toxinidir}/requirements/celery44.txt
+    celery50: celery>=5.0,<6.0
```
Comment (jmbowman) [inline tox.ini:13]: Ideally we'd also have a `celery50.txt`, but I suppose we can defer that for now.

**Review by jmbowman — APPROVED** [review]

**Pull request merged**